File: web_file/src/app/models/file-info.ts
import { User } from "./user";

export interface FileInfo{
    id?:number
    name?: string;
    size?: number;
    user?:User;
    downloadLink?:string;
    date?:string;
    blocked?:boolean;
    dateTime?:Date;
    fileType?:string;
}

File: demo/src/main/java/com/ahsan/file/app/model/response/FileInfo.java
package com.ahsan.file.app.model.response;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Data
public class FileInfo {
    private long id;
    private String name;
    private long size;
    private String downloadLink;
    private String userName;
    private String fileType;
    private boolean isBlocked;
    private LocalDateTime dateTime;
}
